src/app/containers/PodcastPromo/index.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { useContext } from 'react';
+import path from 'ramda/src/path';
+import { string } from 'prop-types';
+import PodcastPromo from '@bbc/psammead-podcast-promo';
+
+import { ServiceContext } from '#contexts/ServiceContext';
+import ImageWithPlaceholder from '#containers/ImageWithPlaceholder';
+
+const Promo = ({
+  title,
+  brandTitle,
+  brandDescription,
+  imageSrc,
+  imageAlt,
+  linkHref,
+  linkText,
+}) => {
+  const { podcastPromo, script, service } = useContext(ServiceContext);
+  const podcastPromoTitle = title || path(['title'], podcastPromo);
+  const podcastBrandTitle = brandTitle || path(['brandTitle'], podcastPromo);
+  const description =
+    brandDescription || path(['brandDescription'], podcastPromo);
+  const img = imageSrc || path(['image', 'src'], podcastPromo);
+  const alt = imageAlt || path(['image', 'alt'], podcastPromo);
+  const url = linkHref || path(['linkLabel', 'href'], podcastPromo);
+  const label = linkText || path(['linkLabel', 'text'], podcastPromo);
+
+  const showPromo = [
+    podcastBrandTitle,
+    podcastPromoTitle,
+    description,
+    img,
+    alt,
+    url,
+    label,
+  ].every(Boolean);
+  if (!showPromo) {
+    return null;
+  }
+
+  return (
+    <div>
+      <PodcastPromo
+        script={script}
+        service={service}
+        role="region"
+        aria-labelledby="podcast-promo"
+      >
+        <PodcastPromo.Title id="podcast-promo">
+          {podcastPromoTitle}
+        </PodcastPromo.Title>
+        <PodcastPromo.Card>
+          <PodcastPromo.Card.ImageWrapper>
+            <ImageWithPlaceholder
+              src={img}
+              alt={alt}
+              height={1}
+              width={1}
+              ratio={100}
+              lazyLoad
+            />
+          </PodcastPromo.Card.ImageWrapper>
+          <PodcastPromo.Card.Content>
+            <PodcastPromo.Card.Title>
+              <PodcastPromo.Card.Link href={url}>
+                <span className="podcast-promo--hover podcast-promo--focus podcast-promo--visited">
+                  {podcastBrandTitle}
+                </span>
+              </PodcastPromo.Card.Link>
+            </PodcastPromo.Card.Title>
+            <PodcastPromo.Card.Description>
+              {description}
+            </PodcastPromo.Card.Description>
+            <PodcastPromo.Card.EpisodesText>
+              {label}
+            </PodcastPromo.Card.EpisodesText>
+          </PodcastPromo.Card.Content>
+        </PodcastPromo.Card>
+      </PodcastPromo>
+    </div>
+  );
+};
+
+Promo.propTypes = {
+  title: string,
```

Comment:
these are all required aren't they? If so then they wouldn't need the default props

Reply:
They actually aren't - G added this to allow you to pass these as props to the component, but if you don't (as is the implementation on the STY page in this PR - [link](https://github.com/bbc/simorgh/pull/8571/files#diff-f6cec37e9ae80823bc21d6ab627f3ed94a396b1c2f0893cbfa03967ecf5824a4R351)), then it accesses `podcastPromo` from the service context, and attempts to use that data 

```js
 const { podcastPromo, script, service } = useContext(ServiceContext);
 const podcastPromoTitle = title || path(['title'], podcastPromo);
```

If neither exist - it returns null

Reply:
Are we not already accessing the ServiceContext in the page component. Could it have been done there instead?

Reply:
Yea good point, will let @pharingee respond

Reply:
@jroebu14 like @RichardPK mentioned, the intention was to change the StoryPage as little as possible since this is seen  more as a trial than an actual feature. I think it might be a good idea to remove the props altogether and keep the data in the context. That way we can use the promo anywhere else and then at a later time extend the props when we need to.

But I think it's better to fetch the ServiceContext data from the component itself. That way it's easier to put the promo on other pages. 

I built the container in a way that can be moved to different pages easily since we're sort of in test mode

Reply:
OK cool. Removing props sounds sensible then. No point in supporting both things when we use one at the moment.